### DISCUSSION TITLE
Changing user-level permissions as part of Global Options

### DISCUSSION THREAD

**Discussion by ofadam** [magicbug/Cloudlog#755] [Ideas / Feature Requests]
As it stands, there are the following user levels in Cloudlog:

Anonymous
Viewer
Editor
API
Administrator

How difficult would it be to be able to assign feature privileges to each level through a panel in Global Options? For example, one might want the Anonymous level to be able to view the full logbook. Or allow the viewer level to see statistics.

**Comment by magicbug** [comment]
Currently its a bit of an arse, however plan is to redefine the roles and how they function so it can come under that change in the new year.
